apps/console/src/app/[entity]/knowledge/render.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import { AppTopbar, KnowledgeBaseView, PageBase } from "@instill-ai/toolkit";
+import { useAppAccessToken } from "lib/use-app-access-token";
+import { useAppTrackToken } from "lib/useAppTrackToken";
+import { useRouter } from "next/navigation";
+
+export const KnowladgeBasePageRender = () => {
+  const router = useRouter();
+
+  const accessToken = useAppAccessToken();
+  useAppTrackToken({ enabled: true });
+  // useInitAmplitude();
+
+  return (
+    <PageBase>
+      <AppTopbar />
+      <PageBase.Container>
+        <PageBase.Content contentPadding="!p-0">
+          <KnowledgeBaseView
+            accessToken={accessToken.isSuccess ? accessToken.data : null}
+            enableQuery={accessToken.isSuccess}
+            router={router}
+          />
```

Comment:
@thewbuk We don't use this kind of props to input access token and enabledQuery now. We can use the state in the useInstillStore. Please take a look at how we make it in other pages